dev-qt/prison/prison-5.38.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A barcode api"
+DESCRIPTION="prison has a prison::AbstractBarcode, which is the base class  \
+for the actual barcode generators, currently prison::QRCodeBarcode and \
+prison::DataMatrixBarcode is the two implemented barcode generators."
+HOMEPAGE="https://api.kde.org/frameworks/prison/html/index.html"
+COPYRIGHT="2010-2017 KDE Foundations"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/frameworks/5.38/prison-5.38.0.tar.xz"
```

Comment:
Thanks @localanu for your PR!

* We like to have a blank line before ARCHITECTURES.
(Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering))
* We prefer to omit SOURCE_DIR when its default value is correct, and that's the case here :)

```diff
-SOURCE_URI="https://download.kde.org/stable/frameworks/5.38/prison-5.38.0.tar.xz"
+SOURCE_URI="https://download.kde.org/stable/frameworks/${portVersion%.*}/prison-$portVersion.tar.xz"
-SOURCE_DIR="prison-5.38.0"
 CHECKSUM_SHA256="95bc65c9164b8ed8cace08be7dfdf12cc9a398a42e2b4c66852faaa1cd1f325f"
+
 ARCHITECTURES="x86 x86_64"
```

Reply:
can I change to 5.39?,no 5.41?
why?,qt5 and ecm version is lower than 5.41?

Reply:
I noticed that prison 5.41.0 requires extra_cmake_modues 5.41.0 (or later).
But we don't have that yet. I created a PR for that: #1914.
What you can do, until my PR gets merged, is use prison 5.39.0.
In case my PR gets merged before the polishing of prison is finished, then you can update to 5.41.0.
The polishing I see is:
* creating a _devel sub-package.
* deciding which files should go to the devel package, and which should stay in the base package.
* use variables from haikuporter (or calls to finddir) instead of hardcoded paths.